project.mml

```diff
@@ -880,12 +893,51 @@ Layer:
                   ELSE 'no'
                 END AS link,
                 COALESCE(layer,0) AS layernotnull,
-                z_order
+                z_order,
+                CASE
+                  WHEN highway IN ('service', 'tertiary_link') THEN 'tertiary_link'
+                  WHEN highway IN ('residential', 'unclassified', 'tertiary') THEN 'white'
+                  ELSE highway
```

Comment:
Mixing "color" white and "class" tertiary_link feels odd.

Reply:
I agree, it’s indeed ugly.